NetCore/LoadResizeSaveParallel.cs

```diff
@@ -27,71 +24,39 @@ public LoadResizeSaveParallel()
                     throw new FileNotFoundException("Could not find an image directory.");
                 }
             }
+
             imageDirectory = Path.Combine(imageDirectory, "images");
+
             // Get at most 20 images from there
-            _images = Directory.EnumerateFiles(imageDirectory).Take(20);
+            this.images = Directory.EnumerateFiles(imageDirectory).Take(20);
+
             // Create the output directory next to the images directory
-            _outputDirectory = Path.Combine(Path.GetDirectoryName(imageDirectory), "output");
-            if (!Directory.Exists(_outputDirectory))
+            this.outputDirectory = Path.Combine(Path.GetDirectoryName(imageDirectory), "output");
+            if (!Directory.Exists(this.outputDirectory))
             {
-                Directory.CreateDirectory(_outputDirectory);
+                Directory.CreateDirectory(this.outputDirectory);
             }
         }
 
         [Benchmark(Baseline = true, Description = "System.Drawing Load, Resize, Save - Parallel")]
-        public void SystemDrawingResizeBenchmark()
-        {
-            Parallel.ForEach(_images, image => {
-                LoadResizeSave.SystemDrawingResize(image, ThumbnailSize, _outputDirectory);
-            });
-        }
+        public void SystemDrawingResizeBenchmark() => Parallel.ForEach(this.images, image => LoadResizeSave.SystemDrawingResize(image, ThumbnailSize, this.outputDirectory));
```

Comment:
Yeah, if you could fix those and put them back on multiple lines, that would make the whole "fix all the formatting" more palatable :D